static/fonts/flaticon/_flaticon.scss

```diff
@@ -0,0 +1,87 @@
+    /*
```

Comment:
Please fix indent and move variables to another file

Reply:
No, an icon pack will be replaced a lot of times. There is no sense to moving variables to another file on every new icon added (adding new icon generates totally new files)

Reply:
Ok I see so we can merge your changes :)